docs/installation/octopus-in-container/octopus-server-container-linux.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Octopus Deploy Server Container Linux
+description: An Octopus Deploy Server instance can be run directly from within a container.
+position: 1
+---
+
+If you want to run an Octopus Deploy Windows container, please refer to the [Octopus Deploy Server Container Windows](/docs/installation/octopus-in-container/octopus-server-container-windows.md) documentation.
+
+Running the Octopus Deploy Server inside a container provides a simple way to set up an Octopus Deploy instance, and upgrading to the latest version of Octopus is just a matter of running a new container with the new image version.
+
+Although there are a few different configuration options, the following is a simple example of starting an Octopus Deploy server container:
+
+```Bash
+$ docker run --interactive 
+             --detach
+             --name OctopusDeploy \
+             --publish 1322:8080 \
+             --env ACCEPT_EULA="Y" \
+             --env DB_CONNECTION_STRING="..." \
+             octopusdeploy/octopusdeploy:2019.13.4
+```
+
+We run in detached mode with `--detach` to allow the container to run in the background.
+
+The `--interactive` argument ensures that `STDIN` is kept open which is required since internally this is what the running `Octopus.Server.exe` process is waiting on to close.
+
+Setting `--name OctopusServer` gives us an easy to remember name for this container. This is optional, but we recommend you provide a name that is meaningful to you, as that will make it easier to perform actions on the container later if necessary.
+
+Using `--publish 1322:8080` maps the _container port_ `8080` to `1322` on the host so that the Octopus instance is accessible outside this sever.
+
+To set the connection string we provide an _environment variable_ `DB_CONNECTION_STRING` (this can be to a local database or an external database).
+
+In this example, we are running the image `octopusdeploy/octopusdeploy:2019.13.4`. The tag maps directly to the Octopus Server version that is bundled inside the image.
+
+## Configuration
+
+When running an Octopus Server Image, the following values can be provided to configure the running Octopus Server instance.
+
+### Environment Variables
+
+Read the Docker [docs](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file) about setting environment variables.
+
+|  Name       |    |
+| ------------- | ------- |
+|**DB_CONNECTION_STRING**|Connection string to the database to use|
+|**MASTER_KEY**|The master key to use to connect to an existing database. If not supplied, and the database does not exist, it will generate a new one. The master key is mandatory if the database exists|
+|**OCTOPUS_SERVER_BASE64_LICENSE**|Your License key for Octopus Deploy. If left empty, it will try and create a free license key for use
```

Comment:
```suggestion
|**OCTOPUS_SERVER_BASE64_LICENSE**|Your license key for Octopus Deploy. If left empty, it will try and create a free license key for use
```